### PR TITLE
Subjects, Semester, Lesson with some bugs

### PR DESCRIPTION
**Pull request by zema2906**
Yarik`s part

### CODE REVIEW

File: src/main/java/com/peter/kist/controller/LessonController.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@Slf4j
+@RequiredArgsConstructor(onConstructor_={@Autowired})
```
Comment (PPrydorozhnyi) [inline src/main/java/com/peter/kist/controller/LessonController.java:17]: change to @RequiredArgsConstructor(onConstructor = @__(@Autowired))

File: src/main/java/com/peter/kist/controller/LessonController.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        Lesson lesson = new Lesson();
+
+        model.addAttribute("lessonForm", lesson);
+
+        return "lessonCreation";
+    }
+
+    @PostMapping("/create")
+    public String createLesson(@ModelAttribute("lessonForm") Lesson lessonForm, BindingResult bindingResult) {
+        //userValidator.validate(userForm, bindingResult);
```
Comment (PPrydorozhnyi) [inline src/main/java/com/peter/kist/controller/LessonController.java:49]: delete all comments

File: src/main/java/com/peter/kist/controller/LessonKindController.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@Slf4j
+@RequiredArgsConstructor(onConstructor_={@Autowired})
```
Comment (PPrydorozhnyi) [inline src/main/java/com/peter/kist/controller/LessonKindController.java:17]: same as in previous controller. same for all controllers

File: src/main/java/com/peter/kist/model/entity/Lesson.java [hunk 4]
```diff
@@ -24,4 +25,14 @@
...
+    @Column(name = "hours")
+    private Integer hours;
+//
+//    @JoinTable(
+//            name = "pr_subject",
+//            joinColumns = {@JoinColumn(name = "student_id")},
+//            inverseJoinColumns = {@JoinColumn(name = "mark_id")}
+//    )
+//    private List<Mark> marks;
+
```
Comment (PPrydorozhnyi) [inline src/main/java/com/peter/kist/model/entity/Lesson.java:37]: remove comments everywhere

File: src/main/webapp/lessonCreation.jsp [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            </div>
+        </spring:bind>
+
+<%--        &lt;%&ndash;subject_name&ndash;%&gt;--%>
+<%--        <spring:bind path="subjectName">--%>
+<%--            <div class="form-group ${status.error ? 'has-error' : ''}">--%>
+<%--                <form:input type="text" path="subjectName" class="form-control" placeholder="Subject Name"/>--%>
+<%--                <form:errors path="subjectName"/>--%>
+<%--            </div>--%>
+<%--        </spring:bind>--%>
```
Comment (PPrydorozhnyi) [inline src/main/webapp/lessonCreation.jsp:60]: see above

**Review by PPrydorozhnyi — CHANGES_REQUESTED** [review]

**Comment by zema2906** [comment]
Ez, done!

**Pull request closed without merging**